nixos/modules/services/admin/realmd.nix

```
@@ -0,0 +1,49 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
```

```suggestion
let
  inherit (lib) mkIf mkOption types;
```

Using `with lib;` is not recommended: https://nix.dev/guides/best-practices.html#with-scopes

I recommend you to use this notation instead :)

---

why was this marked resolved, the suggestion was not included and there's no reasoning against it.